packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -1724,11 +1724,3 @@ describe('ReactSuspenseWithNoopRenderer', () => {
     expect(ReactNoop.getChildren()).toEqual([span('Loading...')]);
   });
 });
-
-// TODO:
```

Comment:
This is unrelated. I happened to notice them now. Deleting these todos because they are outdated.